### PR TITLE
upgrade status-go

### PR DESCRIPTION
**Pull request by cammellos**
Upgrade status-go with changes to log datasync.

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

**Review by yenda — APPROVED** [review]
if the account pr is merged first please don't change status-go version because it will be a more recent one

**Comment by Serhy** [comment]
@cammellos should it be tested with DataSync enabled? 
Am I right that testing scope here is that nothing should be changed with messaging in chats so a regression test required here?

**Comment by cammellos** [comment]
yes, tested with datasync enabled and light regression tests in messaging, thanks!

**Comment by Serhy** [comment]
Tested with DataSync enabled 1-1, Group and public chats. No regression found in this PR. 
The conflicting status-go-version.json as I understand exactly what has been warned in https://github.com/status-im/status-react/pull/8850#pullrequestreview-280700643. If that's the case, @cammellos could you please resolve it and merge?

File: status-go-version.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "_comment": "DO NOT EDIT THIS FILE BY HAND. USE 'scripts/update-status-go.sh <tag>' instead",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "develop",
-    "commit-sha1": "0b403ce25bf25be672f9b39ba97384cb2b72b442",
-    "src-sha256": "0i85yvkbdjd6srkd62b0y03vw67770db4qj53vrh1n8jq6lyqhac"
+    "version": "8dd932f75792db66348e4314a9bd107edf6761a0",
```
Comment (yenda) [inline status-go-version.json]: develop

**Review by yenda — CHANGES_REQUESTED** [review]
just putting a request for change hoping nobody merges that PR before the status-go PR is merged in develop

**Comment by cammellos** [comment]
@yenda could you review again, all the changes have now been added to develop?

**Review by yenda — APPROVED** [review]

**Pull request merged**